demo/src/polyfills.ts

```diff
@@ -1,14 +1,37 @@
-import 'core-js/shim';
+/** IE9, IE10 and IE11 requires all of the following polyfills. **/
+import 'core-js/es6/symbol';
+import 'core-js/es6/object';
+import 'core-js/es6/function';
+import 'core-js/es6/parse-int';
+import 'core-js/es6/parse-float';
+import 'core-js/es6/number';
+import 'core-js/es6/math';
+import 'core-js/es6/string';
+import 'core-js/es6/date';
+import 'core-js/es6/array';
+import 'core-js/es6/regexp';
+import 'core-js/es6/map';
+import 'core-js/es6/set';
 
-require('zone.js/dist/zone');
+/** Evergreen browsers require these. **/
+import 'core-js/es6/reflect';
+import 'core-js/es7/reflect';
+
+/** Zone JS is required by Angular itself. **/
+import 'zone.js/dist/zone';
+
+declare var require: any;
+declare var process: any;
 
 if (process.env.ENV === 'build') {
   // Production
 
 } else {
   // Development
-
   Error['stackTraceLimit'] = Infinity;
-
   require('zone.js/dist/long-stack-trace-zone');
+  require('zone.js/dist/proxy.js');
+  require('zone.js/dist/sync-test');
+  require('zone.js/dist/async-test');
+  require('zone.js/dist/fake-async-test');
```

Comment:
I don't think we use those for unit tests so we shouldn't be having this change.